README.md

```diff
@@ -123,22 +123,7 @@ the [Shimmer Testnet](https://api.testnet.shimmer.network), and retrieves the no
 calling [`Client.get_info()`](https://docs.rs/iota-sdk/latest/iota_sdk/client/core/struct.Client.html#method.get_info),
 and then print the node's information.
 
-```rust
-use iota_sdk::client::{Client, Result};
-
-#[tokio::main]
-async fn main() -> Result<()> {
-    let client = Client::builder()
-        .with_node("https://api.testnet.shimmer.network")? // Insert your node URL here
-        .finish()
-        .await?;
-
-    let info = client.get_info().await?;
-    println!("Node Info: {info:?}");
-
-    Ok(())
-}
-```
+[sdk/examples/client/getting_started.rs](sdk/examples/client/getting_started.rs)
```

Comment:
Do relative links actually work?

Reply:
seems like those links work

Reply:
Yes, we also use them already like here <https://github.com/iotaledger/iota-sdk/blob/develop/README.md#examples> the `examples`